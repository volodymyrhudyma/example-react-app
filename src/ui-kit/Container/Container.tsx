import React, { ReactNode } from 'react';
import { Wrapper } from './styled/Container.styled';

interface IContainerProps {
  children: ReactNode;
}

const Container = ({ children }: IContainerProps) => (
  <Wrapper>{children}</Wrapper>
);

export default Container;
