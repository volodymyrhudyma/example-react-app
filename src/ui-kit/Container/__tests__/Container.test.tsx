import React from 'react';
import { shallow } from 'enzyme';
import Container from '../Container';

describe('Container component', () => {
  it('should render correctly', () => {
    const component = shallow(<Container>Container</Container>);
    expect(component).toMatchSnapshot();
  });
});
