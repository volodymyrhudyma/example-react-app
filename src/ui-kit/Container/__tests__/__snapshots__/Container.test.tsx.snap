// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container component should render correctly 1`] = `
<styled.div>
  Container
</styled.div>
`;
